tests/lib/snaps/test-snapd-desktop-file-ids/meta/snap.yaml

```diff
@@ -14,4 +14,4 @@ apps:
 plugs:
   desktop:
     desktop-file-ids:
-    - org.example
+    - org.example.desktop
```

Comment:
> but let me know if you prefer to just add multiple desktop IDs format to the same test.

yes, that would be enough I believe, no need to duplicate the test. Also, would be interesting to see how a mix of ids would behave.

Thanks for suggesting that!